cmd/admin-revoker/main.go

```diff
@@ -79,7 +79,13 @@ func setupContext(context *cli.Context) (rpc.CertificateAuthorityClient, *blog.A
 	dbMap, err := sa.NewDbMap(c.Revoker.DBDriver, c.Revoker.DBConnect)
 	cmd.FailOnError(err, "Couldn't setup database connection")
 
-	return cac, auditlogger, dbMap
+	saRPC, err := rpc.NewAmqpRPCClient("CA->SA", c.AMQP.SA.Server, ch)
```

Comment:
The name of the outbound queue here should be unique to the (admin revoker, SA) pair, so something like "AR->SA" or (my preference) "AdminRevoker->SA".